lib/Album/AlbumMapper.php

```diff
@@ -332,6 +332,40 @@ public function getCollaborators(int $albumId): array {
 		return array_values(array_filter($collaborators, fn ($c) => $c !== null));
 	}
 
+
+	/**
+	 * @param int $albumId
+	 * @param string $userId
+	 * @return bool
+	 */
+	public function isCollaborator(int $albumId, string $userId): bool {
+		$query = $this->connection->getQueryBuilder();
+		$query->select("collaborator_id", "collaborator_type")
+			->from("photos_albums_collabs")
+			->where($query->expr()->eq('album_id', $query->createNamedParameter($albumId, IQueryBuilder::PARAM_INT)));
+
+		$rows = $query->executeQuery()->fetchAll();
+
+		foreach ($rows as $row) {
+			switch ($row['collaborator_type']) {
+				case self::TYPE_USER:
+					if (!strcmp($row['collaborator_id'], $userId)) {
```

Comment:
Why not use `===` instead of `strcmp` ?

Reply:
No particular reason, I've seen `strcmp` being used elsewhere in similar situations. Shall I change it?

Reply:
Yes, as it is more readable with `===`.
I only used `strcmp` for comparison during array sorting :).

Reply:
Done :)